src/errors/test-run/templates.js

```diff
@@ -375,4 +371,12 @@ export default {
         Error details:
         ${escapeHtml(err.errMsg)}
     `,
+
+    [TEST_RUN_ERRORS.multipleWindowsModeIsDisabledError]: err => `
+        You disabled multi window mode. Remove the "disable-multiple-windows" run option to use the "${err.methodName}" method.
+    `,
+
+    [TEST_RUN_ERRORS.multipleWindowsModeIsNotAvailableInRemoteBrowserError]: err => `
+        Multi window mode is available only in local browsers. Run the test locally to use the "${err.methodName}" method.
+    `,
```

Comment:
@VasilyStrelyaev

Reply:
```js

     [TEST_RUN_ERRORS.multipleWindowsModeIsDisabledError]: err => `
        Multi window mode is disabled. Remove the '--disable-multiple-windows' CLI flag or set the 'disableMultipleWindows' property to 'false' in the API to use the '${err.methodName}' method.
   `,

     [TEST_RUN_ERRORS.multipleWindowsModeIsNotAvailableInRemoteBrowserError]: err => `
        Multi window mode is supported in local browsers only. Run tests locally to use the '${err.methodName}' method.
    `,

Reply:
Checked with Dirk

Reply:
We have `disableMultipleWindows` as option of the `run` method of `runner`